PSReadLine/Completion.cs

```diff
@@ -879,7 +870,6 @@ private void MenuCompleteImpl(Menu menu, CommandCompletion completions)
 
                         if (topAdjustment != 0)
                         {
-                            menu.Top += topAdjustment;
```

Comment:
This PR makes sense to me (given your investigations and the rest of the changes here) except that I can't account for why `menu.Top` no longer needs to be adjusted.

Reply:
Good question! `Top` will be adjusted properly within `menu.DrawMenu`. Adjusting it here will make the `PreviousTop` incorrect within `menu.DrawMenu`.

Reply:
Thanks for the follow-up!